packages/cf-component-link/src/index.js

```diff
@@ -1,3 +1,11 @@
-import Link from './Link';
+import LinkUnstyled from './Link';
+import LinkTheme from './LinkTheme';
+import { applyTheme } from 'cf-style-container';
 
-export default Link;
+const Link = applyTheme(LinkUnstyled, LinkTheme);
+
+export default {
```

Comment:
Yeah I think that's why CI is failing. Once I've merged the other PRs I will come back to this.